docs/pages/tags.html

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Is there an example of how these tags are to be used?

Reply:
We're not using it at the moment. If we decide to use it, it'll simply provide an index for all the tags defined on the site.

Reply:
Alrighty.